packages/solarwinds-apm/CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-# `solarwinds-apm` Changelog
+# 14.0.0
 
-## 14.0.0 (prerelease)
+## Breaking changes
 
-This is a breaking release and a completely new version built from scratch using OpenTelemetry.
-
-- The library now requires the `@opentelemetry/api` package to be installed alongside it, with the version range `^1.3.0`.
+- This is a completely new version of the instrumentation library built upon OpenTelemetry.
+- The library now requires the `@opentelemetry/api` package to be installed alongside it.
```

Comment:
Is there any min/max version of the API required? Or any version is fine?

Reply:
Woops you're right, adding it back.